frontend/src/metabase/containers/ItemPicker/Item.styled.tsx

```diff
@@ -53,6 +61,9 @@ export const ExpandButton = styled(IconButtonWrapper)<{ canSelect: boolean }>`
   border: 1px solid ${color("border")};
 
   &:hover {
+    & svg.Icon {
```

Comment:
We don't use HTML element selectors if possible.

Instead, the `Icon` component should be imported in this file and get `styled`, like this:

```tsx
export const ItemIcon = styled(Icon);
```

Then we could do this:

```suggestion
    & ${ItemIcon} {
```

And of course we need to update the import in `Item.tsx`.